vendor/git.wake

```diff
@@ -28,15 +29,20 @@ def buildAs Unit = match (subscribe releaseAs)
             | setPlanStdout logNever
             | runJob
             | getJobStdout
-        stdout
-        | replace `^v|\n.*` ''
-        | replace `-([0-9]{1,3})-` '.\1+'
-        | replace `-` '~'
-        | Pass
-        # The replace accomplish this:
-        # v0.15.0-alpha-4-g6efe8e9-dirty ... for a tag v0.15.0-alpha with +4 commits
-        #  0.15.0~alpha.4+g6efe8e9~dirty ... which sorts correctly for both rpm+deb
-        # ^      ^     ^ ^        ^
+        stdout | replace `^v|\n.*` '' | replace `-([0-9]{1,3})-` '.\1+' | replace `-` '~' | Pass
+
+
+# The replace accomplish this:
+
+
+# v0.15.0-alpha-4-g6efe8e9-dirty ... for a tag v0.15.0-alpha with +4 commits
+
+
+#  0.15.0~alpha.4+g6efe8e9~dirty ... which sorts correctly for both rpm+deb
+
+
+# ^      ^     ^ ^        ^
+
```

Comment:
Something went wrong here.

Reply:
Uhh I think this might be one of the cases where we have to change the code.

These comments are "floating"  since they don't bind to `Pass` (can only bind up/left if no newlines) and the don't bind down to `topic` because there is a newline and they "are" top-level. It maybe be intentional or a quick of the parser but if you put a comment at the end like this they actually get parsed one level up. So these comments are parsed at top-level instead of being nested by 1

Reply:
All right, two issues then.  First, they don't bind where the author expected (that might just be a "move your comments" requirement).  Second, even at the top level, these should be kept together rather than treated as three separate single-line comments.